tests/common.rs

```diff
@@ -0,0 +1,390 @@
+use std::sync::{Arc, Once, RwLock};
+use std::{env, net};
+
+use stderrlog::StdErrLog;
+use tempfile::TempDir;
+
+use serde_json::json;
+#[cfg(feature = "liquid")]
+use serde_json::Value;
+
+#[cfg(not(feature = "liquid"))]
+use bitcoind::{self as noded, BitcoinD as NodeD};
+#[cfg(feature = "liquid")]
+use elementsd::{self as noded, ElementsD as NodeD};
+
+use noded::bitcoincore_rpc::{self, RpcApi};
+
+use electrs::{
```

Comment:
Have you avoided [`electrsd`](https://github.com/RCasatta/electrsd/) because it does not allow enough control on `electrs` internals?